text/delegatable_cdd/README.md

```diff
@@ -0,0 +1,334 @@
+
+# Motivation
+[motivation]: #motivation
+
+Delegatable credentials are useful in saving operational costs of issuance and can also make issuance cheaper. If an organization that wants to issue credentials has only the head of the organization (or the relevant individual) issuing the credential, it becomes a bottleneck. But if the head decides to delegate the issuance rights to its subordinates, then the credential receivers can check that credential comes from a delegatee of the head. In addition, delegation provides privacy to issuers who might want to remain anonymous to verifiers. This [Aries RFC](https://github.com/hyperledger/aries-rfcs/pull/218) goes into more details of delegatable credentials.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The entity delegating the credential issuance rights and identified to the verifier is called the *root issuer* or the *root delegator*. All issuers have a signing and verification key but only the root issuer's verification key is known to the verifier. The linear structure starting at the root issuer and consisting of subsequent issuers involved in delegation is called the *issuer chain*. For reasons explained later, an issuer is either an odd level issuer or an even level issuer depending on its position in the issuer chain. The *root issuer* is always an even level issuer. Similarly, the linear structure starting at the root issuer's credential and consisting of subsequent issuers' credentials is called the *credential chain*. A credential is either an odd level credential or even level credential depending on the issuer; an even level issuer always issues an even level credential and vice versa. Each credentials in the credential chain is a also called a *link* and has a level which is a  monotonically increasing number starting from 1 and has no gaps. The credential (or link) issued by root issuer has level 1, the subsequent link will have level 2 and so on. During delegation (credential issuance), the delegator in addition to issuing a credential, sends its credential chain also to the delgatee. A presentation created from the delegated credential is called *attribute token*.  
```

Comment:
If I read this correctly:

1. the root issuer is always an even level issuer
1. an even level issuer always issues an even level credential
1. The credential issued by the root issuer has level 1

This seems off.

Reply:
Yes, was a typo. Fixed.